.github/workflows/REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,712 @@
+{
+    // do some work
+
+    $this->comment('All ok!');
+}
+```
+
+If possible use a descriptive success message eg. `Old records deleted`.
+
+## Routing
+
+Public-facing urls must use kebab-case.
+
+```
+https://spatie.be/open-source
+https://spatie.be/jobs/front-end-developer
+```
+
+Route names must use camelCase.
+
+```php
+Route::get('open-source', 'OpenSourceController@index')->name('openSource');
+```
+
+```html
+<a href="{{ route('openSource') }}">
+    Open Source
+</a>
+```
+
+All routes have an http verb, that's why we like to put the verb first when defining a route. It makes a group of routes very readable. Any other route options should come after it.
+
+```php
+// good: all http verbs come first
+Route::get('/', 'HomeController@index')->name('home');
+Route::get('open-source', 'OpenSourceController@index')->middleware('openSource');
+
+// bad: http verbs not easily scannable
+Route::name('home')->get('/', 'HomeController@index');
+Route::middleware('openSource')->get('OpenSourceController@index');
+```
+
+Route parameters should use camelCase.
+
+```php
+Route::get('news/{newsItem}', 'NewsItemsController@index');
+```
+
+A route url should not start with `/` unless the url would be an empty string.
+
+```php
+// good
+Route::get('/', 'HomeController@index');
+Route::get('open-source', 'OpenSourceController@index');
+
+//bad
+Route::get('', 'HomeController@index');
+Route::get('/open-source', 'OpenSourceController@index');
+```
+
+## Controllers
+
+Controllers that control a resource must use the plural resource name.
+
+```php
+class PostsController
+{
+    // ...
+}
+```
+
+Try to keep controllers simple and stick to the default CRUD keywords (`index`, `create`, `store`, `show`, `edit`, `update`, `destroy`). Extract a new controller if you need other actions.
+
+In the following example, we could have `PostsController@favorite`, and `PostsController@unfavorite`, or we could extract it to a separate `FavoritePostsController`.
+
+```php
+class PostsController
+{
+    public function create()
+    {
+        // ...
+    }
+
+    // ...
+
+    public function favorite(Post $post)
+    {
+        request()->user()->favorites()->attach($post);
+
+        return response(null, 200);
+    }
+
+    public function unfavorite(Post $post)
+    {
+        request()->user()->favorites()->detach($post);
+
+        return response(null, 200);
+    }
+}
+```
+
+Here we fall back to default CRUD words, `store` and `destroy`.
+
+```php
+class PostsFavoriteController
+class UsersImageController
+class CompaniesLotController
+{
+    public function favorite(Post $post)
+    {
+        request()->user()->favorites()->attach($post);
+
+        return response(null, 200);
+    }
+
+    public function unfavorite(Post $post)
+    {
+        request()->user()->favorites()->detach($post);
+
+        return response(null, 200);
+    }
+    
+   public function mergeLot() {
+   
+   }
+}
+```
+
+This is a loose guideline that doesn't need to be enforced.
+
```

Comment:
Avoid using vague language such as 'This is a loose guideline that doesn't need to be enforced.' It can lead to confusion about whether the guideline should be followed or not.